### PR TITLE
Divyanshu | Stage | Convert Setting side bar all tabs into material

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/85ztg2ujb

### CODE REVIEW

File: apps/web-giddh/src/assets/locale/en.json [hunk 1]
```diff
@@ -456,8 +456,6 @@
     "app_select_branch": "Select Branch",
     "app_voucher": "Voucher",
     "app_alt_c": "Alt+C",
-    "app_alt_d": "Alt+D",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/en.json]: why is this getting deleted?

File: apps/web-giddh/src/assets/locale/hi.json [hunk 2]
```diff
@@ -456,8 +456,6 @@
     "app_select_branch": "शाखा का चयन करें",
     "app_voucher": "वाउचर",
     "app_alt_c": "Alt+C",
-    "app_alt_d": "Alt+D",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/hi.json]: why is this getting deleted?

File: apps/web-giddh/src/assets/locale/journal-voucher/en.json [hunk 3]
```diff
@@ -30,7 +30,5 @@
     "receipt_entry": "Receipt Entry",
     "bill_wise_details": "Bill wise Details for",
     "upto": "Upto",
-    "entry_created": "Entry created successfully",
-    "discount_list": "Discount List",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/journal-voucher/en.json:34]: why is this getting deleted?

File: apps/web-giddh/src/assets/locale/journal-voucher/hi.json [hunk 4]
```diff
@@ -30,7 +30,5 @@
 	"receipt_entry": "रसीद प्रविष्टि",
 	"bill_wise_details": "बिल के अनुसार विवरण",
 	"upto": "तक",
-	"entry_created": "प्रविष्टि सफलतापूर्वक बनाई गई",
-    "discount_list": "छूट सूची",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/journal-voucher/hi.json:34]: why is this getting deleted?

File: apps/web-giddh/src/assets/locale/journal-voucher/mr.json [hunk 5]
```diff
@@ -30,7 +30,5 @@
 	"receipt_entry": "पावती नोंद",
 	"bill_wise_details": "यासाठी बिल निहाय तपशील",
 	"upto": "इथपर्यंत",
-	"entry_created": "प्रविष्टी यशस्वीरित्या तयार केली",
-    "discount_list": "सवलत यादी",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/journal-voucher/mr.json:34]: why is this getting deleted?

File: apps/web-giddh/src/assets/locale/mr.json [hunk 6]
```diff
@@ -456,8 +456,6 @@
     "app_select_branch": "शाखा निवडा",
     "app_voucher": "व्हाउचर",
     "app_alt_c": "Alt+C",
-    "app_alt_d": "Alt+D",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/mr.json]: why is this getting deleted?

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/locale/en.json (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/assets/locale/en.json]: Forgot to add this for JV

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**